ufs/ccpp/data/MED_typedefs.F90

```diff
@@ -73,6 +73,7 @@ module MED_typedefs
     real (kind=kind_phys),pointer :: lake_q2m (:)    => null() !< 2 meter humidity from CLM Lake model
     real(kind=kind_phys), pointer :: wind(:)         => null() !< wind speed at lowest model level (m/s)
     logical,              pointer :: flag_iter(:)    => null() !< flag for iteration
+    logical,              pointer :: lake_freeze(:)  => null() !< flag for lake freeze
```

Comment:
@JiliDong-NOAA Can we change the name to: flag_lakefreeze to have consistent name convention?

Reply:
name changes as suggested.